GoogleForCreators/web-stories-wp
# Templates: Added links to DIY, Fashion, and Fitness

## Summary

This PR adds links to the link pages for our templates.

Here's a list of our templates and which page was updated:
- `Cooking`: This template does _not_ have a links page, no update required.

- `Entertainmnet`: This template does _not_ have a links page, no update required.

- `Travel`:  This template does _not_ have a links page, no update required.

- `Beauty`: Links will be added to its links page in a _separate_ PR.

- `Wellbeing`: Links were added to this template in [another PR](https://github.com/google/web-stories-wp/pull/3018), so no changes were made in this PR.

- `DIY`: Links were added to page 9.
<img src="https://user-images.githubusercontent.com/40646372/86813618-444e4000-c035-11ea-81a3-a9fdf8d52fe4.gif" height="200">

- `Fashion`: Links were added to page 8.
<img src="https://user-images.githubusercontent.com/40646372/86813628-4912f400-c035-11ea-9844-c6f4a2558c80.gif" height="200">

- `Fitness`: Links were added to page 7.
<img src="https://user-images.githubusercontent.com/40646372/86813636-4ca67b00-c035-11ea-97be-2c9afc4858f9.gif" height="200">

## To-do

- The Beauty template will get updated in a separate PR.

## User-facing changes

- The pages list above should have links embedded in them that link to `example.com`.

## Testing Instructions

1.) Go to "Explore Templates".
2.) Click on "Use Template" to create 3 new stories from the `DIY`, `Fashion`, and `Fitness` templates.
3.) Click on "Open in Editor" for each new story and navigate to its links page (DIY - page 9, Fashion - page 8, Fitness - page 7) and hover over the center of each link to see "Example Domain" pop up for each one.

Fixes #2888
